struct.go

```diff
@@ -198,24 +200,34 @@ func (s *Struct) InsertIntoForTag(table string, tag string, value interface{}) *
 		return ib
 	}
 
-	v := dereferencedValue(value)
+	var vs []reflect.Value
```

Comment:
Should use `make` for better performance.

`vs := make([]reflect.Value, 0, len(value))`